.ci/Jenkinsfile

```diff
@@ -173,14 +193,34 @@ def cleanup(){
   unstash 'source'
 }
 
-def withNodeEnv(Map args=[:], Closure body){
+def withNodeInDockerEnv(Map args=[:], Closure body){
   docker.image("${NODE_DOCKER_IMAGE}").inside(" --security-opt seccomp=${SECCOMP_FILE}"){
     withEnv(["HOME=${WORKSPACE}"]) {
       body()
     }
   }
 }
 
+def withNodeEnv(Map args=[:], Closure body){
+  withEnv(["HOME=${WORKSPACE}"]) {
+    sh(label: 'install Node.js', script: '''
+      set +x
+      if [ -z "$(command -v nvm)" ]; then
+        curl -so- https://raw.githubusercontent.com/nvm-sh/nvm/v0.37.1/install.sh | bash
+      fi
+
+      export NVM_DIR="${HOME}/.nvm"
+      [ -s "${NVM_DIR}/nvm.sh" ] && \\. "${NVM_DIR}/nvm.sh"
+      nvm install --lts
```

Comment:
better to test it on the version we use in synthetics image https://github.com/elastic/synthetics/blob/1375d46e600f7cdd273304cdbc5a1741f1d4431e/Dockerfile#L27